src/main/java/Parser.java

```diff
@@ -0,0 +1,133 @@
+import java.time.DateTimeException;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Scanner;
+import tasks.Deadline;
+import tasks.Event;
+import tasks.ToDo;
+import tasks.Task;
+
+public class Parser {
+    public void parse(TaskList tasks) {
+        Scanner echoObject = new Scanner(System.in);
+        String echo = echoObject.nextLine();
+        while (!echo.equals("bye")) {
+            try {
+                if (!echo.equals("list")) {
```

Comment:
Perhaps you could consider using switch statements to handle the different cases?